### DISCUSSION TITLE
What is prompt retriever and why it's useful

### DISCUSSION THREAD

**Discussion by TacioRibeiro** [FlowiseAI/Flowise#676] [General]
Does anyone has an idea ?

**Comment by chungyau97** [comment]
Hi @TacioRibeiro, 

This is a good starting point for understanding the usage of prompt. [LangChain JS Prompt](https://js.langchain.com/docs/modules/model_io/prompts/)

From my understanding, it will manipulate the behavior of bot reply [tones, style, language, ...]
